### PR TITLE
Styling

### PR DESCRIPTION
**Pull request by kev1n80**
updated the projects, photography, and blog section and the size of texts in bigger screens

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -4,48 +4,136 @@
...
+            BAM. The photo was created.
+          </p>
+        </div>
+      </div>  
 
-      <h3>Projects</h3>
-      <a target="_blank" href="https://github.com/kev1n80">Visit my Github!</a>
-      <a target="_blank" href="https://www.linkedin.com/in/kevin-cam">Visit my LinkedIn!</a>
-      <ul>
-          <li>Schedify</li>
```
Comment (hoorayhu) [inline portfolio/src/main/webapp/index.html:34]: Consider using more HTML5 tags, like <article> <details>

File: portfolio/src/main/webapp/style.css [hunk 2]
```diff
@@ -1,18 +1,271 @@
 #content {
   margin-left: auto;
   margin-right: auto;
-  width: 650px;
+  text-align: center;
+  width: 100vw;
```
Comment (hoorayhu) [inline portfolio/src/main/webapp/style.css]: Consider 100% instead of 100vw in most cases.

File: portfolio/src/main/webapp/style.css [hunk 3]
```diff
@@ -1,18 +1,271 @@
...
+#quick-info-content {
+  margin: auto;
+}
+
+#quick-info-content button { 
+  border-radius: 10px;
+  background-color: white;
+}
+
+#quick-info-content button:hover {
```
Comment (hoorayhu) [inline portfolio/src/main/webapp/style.css]: Prefer class names instead of ids in CSS rules for future extensibility.

**Review by hoorayhu — APPROVED** [review]
Good in general. Some minor suggestions to consider.

File: portfolio/src/main/webapp/index.html (hunk 1 above)
Comment (kev1n80) [reply portfolio/src/main/webapp/index.html:34]: I was wondering if there were any in particular that you had in mind.

**Pull request merged**